src/components/survey/__tests__/sorting-hat-reducer.test.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * @jest-environment jsdom
+ */
+
+// could eventually test that cioIdentify and track are being called appropriately
+
+import {cioIdentify} from '../cio-identify'
+import {sortingHatReducer, sortingHatInitialState} from '../sorting-hat-reducer'
+import {track} from 'utils/analytics'
+
+jest.mock('../cio-identify', () => ({
+  cioIdentify: jest.fn(() => null),
+}))
+
+jest.mock('utils/analytics', () => ({
+  track: jest.fn(),
+}))
+
+const defaultFirstQuestion =
+  sortingHatInitialState.data[sortingHatInitialState.currentQuestionKey]
+
+describe('empty action', () => {
+  test('should return the initial state', () => {
+    expect(sortingHatReducer(sortingHatInitialState, {})).toStrictEqual(
+      sortingHatInitialState,
+    )
+  })
```

Comment:
why do you prefer `toStrictEqual` over `toEqual`?

One thing I like to do is explicitly name the item that is the subject of the expect.

```
test('should return the initial state', () => {
    const initialState = sortingHatReducer(sortingHatInitialState, {})
    
    expect(initialState).toStrictEqual(sortingHatInitialState)
  })
```

I also use whitespace to give the test additional readability.

Tests are an area where I feel readability aesthtics are **vital**.